vortex-io/src/tokio.rs

```diff
@@ -81,18 +81,17 @@ impl VortexReadAt for TokioFile {
     }
 
     #[cfg_attr(feature = "tracing", tracing::instrument(skip(self)))]
-    fn size(&self) -> impl Future<Output = u64> + 'static {
+    fn size(&self) -> impl Future<Output = io::Result<u64>> + 'static {
         let this = self.clone();
 
         async move {
-            let res = tokio::task::spawn_blocking(move || {
+            tokio::task::spawn_blocking(move || {
                 this.metadata()
                     .unwrap_or_else(|e| vortex_panic!("access TokioFile metadata: {e}"))
                     .len()
```

Comment:
we can just forward the io::result from this.metadata()

Reply:
Good spot